cmd/lotus-storage-miner/actor.go

```diff
@@ -70,7 +75,10 @@ var actorSetAddrsCmd = &cli.Command{
 		ctx := lcli.ReqContext(cctx)
 
 		var addrs []abi.Multiaddrs
-		for _, a := range cctx.Args().Slice() {
+		for _, a := range args {
```

Comment:
Is this to enable a way to unset all addresses? If so, I'd rather do the following:

- Add a `--unset` boolean flag
- If it's supplied with no arguments, unset the miner addresses (what this currently does)
- If it's supplied with arguments, fail 
- If it isn't supplied, and yet no arguments are provided, fail